rasa/nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -38,13 +38,27 @@ def __init__(
         self.case_sensitive = self.component_config.get("case_sensitive")
         self.intent_keyword_map = intent_keyword_map or {}
 
+    def train_chunk(
+        self,
+        training_data_chunk: TrainingDataChunk,
+        config: Optional[RasaNLUModelConfig] = None,
+        **kwargs: Any,
+    ) -> None:
+        """Train this component on the given chunk.
+
+        See parent class for more information.
+        """
+        raise Exception(
```

Comment:
Would not raise `Exception` here but rather a more specific one. What about creating a `RasaTrainException` in https://github.com/RasaHQ/rasa/blob/master/rasa/shared/exceptions.py, subclassing our `RasaException`?

Reply:
created `RasaTrainChunkException`